pkg/k8s/apis/cilium.io/v2alpha1/flowlog.go

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package v2alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:onlyVerbs=get,list,watch
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:categories={cilium},singular="ciliumflowlog",path="ciliumflowlogs",scope="Cluster",shortName={cfl,ciliumfl}
+// +kubebuilder:printcolumn:JSONPath=".metadata.creationTimestamp",description="Time duration since creation of CiliumFlow Logging",name="Age",type=date
+// +kubebuilder:storageversion
+// +kubebuilder:subresource:status
+
+// CiliumFlowLog configures a flow log task for Cilium agents.
+//
+// +deepequal-gen=false
+type CiliumFlowLog struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+
+	// Spec defines the desired specification/configuration of the flow log.
+	//
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Spec is immutable"
+	Spec FlowLogSpec `json:"spec"`
+
+	// Status defines the realized specification/configuration and status
+	// of the flow log.
+	//
+	// +kubebuilder:validation:Optional
+	Status FlowLogStatus `json:"status,omitempty"`
+}
+
+// FlowLogSpec defines configuration of a flow log task.
+//
+// +deepequal-gen=false
+type FlowLogSpec struct {
+	// List of field names that will be kept in the log output.
```

Comment:
`// List of flowpb.FlowFilter field names`

Are these for the FlowFilter fields? If so, let's make it clear in the comment.

Reply:
They are not FlowFilter fields. They are flowpb.Flow's